pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/Policies.java

```diff
@@ -241,18 +241,16 @@ public String toString() {
     }
 
 
-    private static final long MAX_BUNDLES = ((long) 1) << 32;
-
     public static BundlesData getBundles(int numBundles) {
-        if (numBundles <= 0 || numBundles > MAX_BUNDLES) {
+        if (numBundles <= 0) {
```

Comment:
Please take a look at this change. I am not sure why we used the long value before.

Reply:
I don't think the change is correct. What was the findbugs error here?

Reply:
The findbugs error is: INT_BAD_COMPARISON_WITH_INT_VALUE

Reply:
It should be only affected by the comparison of the values. I remove the comparison between the int value and long value. 
But I still wonder why we use a 'long' to represent the max size of the bundles?